### ISSUE TITLE
Let theme register backgrounds and background images?

### ISSUE THREAD

**Issue by strarsis** [lubusIN/block-background#14]
Like with `add_theme_support( 'editor-color-palette',`, themes should be able to register their own gradients and images for background, too.
And what about complex backgrounds that use wrappers and pseudo-elements?

**Comment by strarsis** [comment]
Apparently this is already in the core and can be set up by theme:
https://developer.wordpress.org/block-editor/developers/themes/theme-support/#block-gradient-presets

**Comment by ajitbohra** [comment]
Yes this is on the list along with few  more updates which are planned for release by end of the month

**Comment by strarsis** [comment]
@ajitbohra: Related: https://github.com/WordPress/gutenberg/issues/16662
Apparently, many blocks still don't support gradient backgrounds.

**Comment by ajitbohra** [comment]
We are working on an update to add the Native gradient picker.

**Comment by strarsis** [comment]
@ajitbohra: This PR in core seems to be similar: https://github.com/WordPress/gutenberg/pull/21375

**Comment by ajitbohra** [comment]
The core is introducing a new hook to add support for colors to block via `block support` currently its experimental. We are planning to use it in the future.
